packages/lib/src/components/PayPal/utils.ts

```diff
@@ -28,6 +28,7 @@ const getPaypalSettings = ({ amount, countryCode, debug, environment = '', local
         ...(debug && isTestEnvironment && { debug }),
         ...(currency && { currency }),
         ...(intent && { intent }),
+        ...{ commit },
```

Comment:
There's no need to use the spread operator here. Just using the shorthand is enough.
```suggestion
        commit,
```

Reply:
Updated